pybamm/install_odes.py

```diff
@@ -41,10 +45,7 @@ def install_sundials(download_dir, install_dir):
         raise RuntimeError("CMake must be installed to build SUNDIALS.")
 
     url = (
-        "https://github.com/LLNL/"
-        + "sundials/releases/download/v{}/sundials-{}.tar.gz".format(
-            sundials_version, sundials_version
-        )
+        f"https://github.com/LLNL/sundials/releases/download/v{sundials_version}/sundials-{sundials_version}.tar.gz"
```

Comment:
An improvement could be that instead of using `sundials_version`, we could provide it as a constant `SUNDIALS_VERSION` at the top of the file (would make it easier to access and edit)

Reply:
Done